app/views/account/fragments/planList.scala.html

```diff
@@ -0,0 +1,34 @@
+@import com.gu.memsub.subsv2.SubscriptionPlan
+@import views.support.Dates._
+@import views.support.Pricing._
+@import com.gu.memsub.Benefit.PaperDay
+@import org.joda.time.LocalDate.now
+
+@(plans: List[SubscriptionPlan.ContentSubscription], singularLabel: String, chosenPaperDays: List[PaperDay] = List.empty)
+
+@if(plans.length > 1) {
+    <dt class="mma-section__list--title">@{
+        singularLabel + "s"
+    }</dt>
+}
+@if(plans.length == 1) {
+    <dt class="mma-section__list--title">@singularLabel</dt>
+}
```

Comment:
I though about that but I don't want to output anything in case the plan list is empty.  I could do that and put everything inside a condition that checks that there are plans in the list